PWGDQ/Core/VarManager.cxx

```diff
@@ -24,6 +24,8 @@ float VarManager::fgValues[VarManager::kNVars] = {0.0f};
 std::map<int, int> VarManager::fgRunMap;
 TString VarManager::fgRunStr = "";
 std::vector<int> VarManager::fgRunList = {0};
+float VarManager::CenterOfMassEnergy = 13600;         // GeV
+float VarManager::MassofCollidingParticle = 9.382720; // GeV
```

Comment:
I would like to use naming conventions for the class data members, so please change these two names such that they start with "fg", similar to the other ones. The "f" means it is a data member, and the "g" means its static. These are rather old conventions, but work very well when reading the code.